src/components/Wizard/CreateVmWizard/NetworksTab.js

```diff
@@ -4,20 +4,36 @@ import { FormFactory } from '../../Form';
 import { TableFactory } from '../../Table/TableFactory';
 import { ACTIONS_TYPE, DELETE_ACTION } from '../../Table/constants';
 import { POD_NETWORK } from '../../../constants';
+import {
+  SELECT_NETWORK,
```

Comment:
Upstream is not internationalized either. In case of changed requirements,it will be quite challenging there.

Anyway, the least we can do here is to isolate the strings as much as possible. I suggest to move them to a specific file, like `strings.js`. I would co-locate these `strings.js` files with components for now.

Reply:
added `strings.js` which now contains all UI strings from `constants.js`